src/index.ts

```diff
@@ -1,9 +1,10 @@
 import type { PlaygroundPlugin, PluginUtils } from "./vendor/playground";
 
 import prettier from "prettier/standalone";
-import tsPlugin from "prettier/parser-typescript";
+import estreePlugin from "prettier/plugins/estree";
+import tsPlugin from "prettier/plugins/typescript";
 
-const plugins = [tsPlugin];
+const plugins = [estreePlugin, tsPlugin];
```

Comment:
according to [the prettier docs](https://prettier.io/docs/en/browser), estree is also necessary to print typescript